### PR TITLE
Fix warnings Pytest

### PR DESCRIPTION
**Pull request by thomasdenecker**
- [x] Add timezone in datetime for `date_joined`
- [x] Add timezone in datetime for `last_login`

### CODE REVIEW

File: apps/core/factories.py [hunk 1]
```diff
@@ -1,10 +1,18 @@
 from factory import Faker, Iterator, SubFactory
 from factory.django import DjangoModelFactory
+from django.utils import timezone
 
 from apps.data.factories import EntryFactory, RepositoryFactory
+from faker import Faker as faker_Faker
 from . import models
 
 
+def Get_date_with_timezone():
```
Comment (jmaupetit) [inline apps/core/factories.py]: Naming convention for functions is lower case with words separated by underscores (see [PEP8 reference](https://www.python.org/dev/peps/pep-0008/#function-names)).

File: apps/core/factories.py [hunk 2]
```diff
@@ -1,10 +1,18 @@
...
+    TZ = timezone.get_default_timezone()
+    fake = faker_Faker()
+    return fake.date_time_this_decade(tzinfo=TZ)
```
Comment (jmaupetit) [inline apps/core/factories.py]: BTW, you don't need this function at all.

File: apps/core/factories.py [hunk 3]
```diff
@@ -1,10 +1,18 @@
 from factory import Faker, Iterator, SubFactory
 from factory.django import DjangoModelFactory
+from django.utils import timezone
```
Comment (jmaupetit) [inline apps/core/factories.py]:
```python
from django.utils.timezone import get_default_timezone
```

File: apps/core/factories.py [hunk 4]
```diff
@@ -1,10 +1,18 @@
...
 
 from apps.data.factories import EntryFactory, RepositoryFactory
+from faker import Faker as faker_Faker
```
Comment (jmaupetit) [inline apps/core/factories.py]: You don't need to re-import Faker, it's already imported via `factory` (see line 1).

File: apps/core/factories.py [hunk 5]
```diff
@@ -53,13 +61,13 @@ class Meta:
 
 class PixelerFactory(DjangoModelFactory):
 
-    date_joined = Faker('date_time_this_decade')
+    date_joined = Get_date_with_timezone()
```
Comment (jmaupetit) [inline apps/core/factories.py]: Use instead:

```python
date_joined = Faker('date_time_this_decade', tzinfo=get_default_timezone())
```

File: apps/core/factories.py [hunk 6]
```diff
@@ -53,13 +61,13 @@ class Meta:
...
     email = Faker('email')
     first_name = Faker('first_name')
     is_active = Faker('pybool')
     is_staff = Faker('pybool')
     is_superuser = Faker('pybool')
-    last_login = Faker('date_time_this_decade')
+    last_login = Get_date_with_timezone()
```
Comment (jmaupetit) [inline apps/core/factories.py]:
```python
last_login = Faker('date_time_this_decade', tzinfo=get_default_timezone())
```

File: apps/core/factories.py (hunk 1 above)
Comment (thomasdenecker) [reply apps/core/factories.py]: Ok. Thank you. Why didn't flake8 say anything? Strange...

**Comment by thomasdenecker** [comment]
Why make it simple when we can make it complicated? :blush:
All points are corrected. Thank you for your correction.

**Pull request merged**